apps/meteor/client/views/omnichannel/analytics/DateRangePicker.tsx

```diff
@@ -39,12 +51,12 @@ const DateRangePicker = ({ onChange = () => {}, ...props }) => {
 			start: range.start,
 		};
 		setRange(rangeObj);
-		onChange(rangeObj);
+		onChange?.(rangeObj);
```

Comment:
onChange doesn't look optional, is this chaining still needed?

Reply:
Nope, I forget to change haha. Sorry and thank you